### ISSUE TITLE
feature request: support for embedded tilesets

### ISSUE THREAD

**Issue by monogamefan** [TheBoneJarmer/TiledCS#57]
Hi

I would like to request support for embedded tilesets.
I can't get it to work with tiledCS for now :(

**Comment by TheBoneJarmer** [comment]
You are correct, it is not. TiledCS only supports external tilesets for now. I will add support in the next version. :)
